### ISSUE TITLE
quotes not applied to all properties of an object

### ISSUE THREAD

**Issue by v2kiran** [nightroman/PsdKit#1]
```powershell
PS C:\> gsv bits | select name,status,starttype | ConvertTo-Psd
@{
    Name = 'bits'
    Status = Running
    StartType = Automatic
}
PS C:\>

```

**Comment by nightroman** [comment]
Thank you for reporting this. Hmm, these are enums and I did not cover them yet. And I am surprised that it worked this way, too (I would expect an error instead). I will investigate and think of this case anyway.

**Comment by nightroman** [comment]
The interesting question is how we should write enums, as just quoted strings or with casts to their enum types.

**Comment by nightroman** [comment]
Investigated.

Why it works in this way and does not write an error. Because the checked type
code of enum is `Int32` and it is treated as a number (incorrectly), i.e.
written without quotes. This should be fixed.

How we should write enums. There are two reasonable ways:

1. As strings, i.e. `'Cyan'`.
    - Pros: concise; import always works
    - Cons: the enum type is lost
2. With type casts, i.e. `[System.ConsoleColor] 'Cyan'`.
    - Pros: the enum type is preserved
    - Cons: verbose; may need some assemblies preloaded for import

Your case with `Get-Service` and its `[System.ServiceProcess.ServiceControllerStatus]`
(i.e. `'Running'`) is a good example of way 2 problems. On importing in a clean
session, the enum type is not yet available and importing fails.

For now (v0 series), I am going to use the way 1. Lost enum types is a less
serious problem than potential inability to read data back. In your case,
it is not even obvious what assembly should be preloaded for enabling the
type.

**Comment by nightroman** [comment]
Fixed in v0.0.3
I think eventually we will need some options specifying how particular types are written. I am collecting my own experience and waiting for some feedback. Thank you very much for yours.

**Comment by v2kiran** [comment]
wow thanks for the quick response!...I didn't even realize that they were enums until you mentioned it and the explanation of the 2 props being treated as numbers makes sense.
I just provided that as an example and don`t have a real world case at least not yet :) but thanks for incorporating the change.
